packages/core/src/types/value.ts

```diff
@@ -223,7 +223,7 @@ export interface IHMatrix<T> {
   dy: T;
 }
 
-export type Color<T> = IRGBA<T> | IHSVA<T>;
+export type Color<T> = IRGBA<T> | IHSVA<T> | INONE<T>;
```

Comment:
A better name is perhaps `INone` or `INoPaint`. Also the `<T>` generics is probably not needed here. Some of the value types are parametrized mainly because of autodiff. See [autodiff guide](https://github.com/penrose/penrose/wiki/Autodiff-guide) for why.

Reply:
Renamed & de-parameterized type in updated PR.